src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -719,13 +719,17 @@ public static EntityTeleportEvent.EnderEntity onEnderTeleport(LivingEntity entit
         return event;
     }
 
+    /**
+     * @deprecated Use {@link #onEnderPearlLand(ServerPlayer, double, double, double, ThrownEnderpearl, float, HitResult) the hit result-sensitive version}
```

Comment:
_Technically_, this should use `{@linkplain}`.
```suggestion
     * @deprecated Use {@linkplain #onEnderPearlLand(ServerPlayer, double, double, double, ThrownEnderpearl, float, HitResult) the hit result-sensitive version}.
```